test-harness/index.js

```diff
@@ -0,0 +1,270 @@
+const fetch = require('node-fetch');
+const requests = require('./requests');
+
+const { exec } = require('child_process');
+const { makeRequest, constructMasterFileName, readFile } = require('./helpers');
+
+async function waitForPrism(done) {
```

Comment:
For this kind of operations is usually better to attach yourself to the `stdout` stream you can get from the `exec` function and check when the output is coming.

Reply:
@XVincentX, I think this is good for now, we can always change it when it turns out that the current implementation is not good enough